### ISSUE TITLE
Hide comment delete/report links in mobile view

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-www#2727]
Per design from @kathymakes, delete and report links should be taken out from comments in mobile stacking view.

This is a follow-up to https://github.com/LLK/scratch-www/pull/2512

**Comment by towerofnix** [comment]
To be clear, does this mean removing the ability to report comments on mobile devices?

**Comment by benjiwheeler** [comment]
@towerofnix Yes (well, widths < 768px... so many tablets will still give you the full experience in landscape.)

Do you think the ability to report comments on mobile should stay?

**Comment by towerofnix** [comment]
Yes, I do. Now that Scratch will be capable of playing projects on small mobile devices, there will be more users who use Scratch on phones in general. I understand that mobile devices are harder to design for because they have much less screen estate to work with, but being able to report comments is a core part of being part of the Scratch community. So, while maybe replying to comments (that isn't implemented on mobile view, right?) is still a little ways off before it's added on mobile, we certainly should not make it seem as though the mobile view is being *ignored* by removing the basic delete/report buttons.

Btw, there's a simple logic that if comments *can* be read on phones, they *will* be read on phones. (Sort of inherent given how big the Scratch community, but especially true because people will be more likely to browse comments on phones since they can run projects on phones too.) And so Community Guidelines-breaking comments will be read on phones too - and without a report button, those comments will be left there, to cause harm longer than if there were a report button available to the person who read it on their phone.
